man/generateRWrappers.Rd

```diff
@@ -0,0 +1,144 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/PythonPkgWrapperUtils.R
+\name{generateRWrappers}
+\alias{generateRWrappers}
+\title{Generate R wrappers for Python classes and functions}
+\usage{
+generateRWrappers(pyPkg, container, setGenericCallback, functionFilter = NULL,
+  classFilter = NULL, functionPrefix = NULL, pySingletonName = NULL,
+  transformReturnObject = NULL)
+}
+\arguments{
+\item{pyPkg}{The Python package name}
+
+\item{container}{The fully qualified name of a Python module or a Python class to be wrapped}
+
+\item{setGenericCallback}{The callback to setGeneric defined in the target R package}
+
+\item{functionFilter}{Optional function to intercept and modify the auto-generated function metadata.}
+
+\item{classFilter}{Optional function to intercept and modify the auto-generated class metadata.}
+
+\item{functionPrefix}{Optional text to add to the name of the wrapped functions.}
+
+\item{pySingletonName}{Optional parameter used to expose a set Python functions which are an object's
+methods, but without exposing the object itself. If the \code{container} parameter is a class then this must
+be the name of a Python variable referencing an instance of the class. Otherwise, this must be NULL.
+See example 4.}
+
+\item{transformReturnObject}{Optional function to change returned values in R.}
+}
+\description{
+This function generates R wrappers for Python classes and functions
+in the given Python container
+}
+\details{
+\itemize{
+\item \code{container} can take the same value as \code{pyPkg}, can be a module or class within the Python package.
+\item \code{setGeneric} function must be defined in the same environment that \code{generateRWrappers}
+is called. See example 1.
+\item \code{functionFilter} and \code{classFilter} are optional functions defined by the caller.
+\item \code{functionFilter} takes as input the metadata for a generated function and either modifies it
+or returns NULL to omit it from the set of generated functions. The metadata object is a list
+having fields:\preformatted{'name': character
+'args': named list having fields:
+    'args': a list of the argument names (it may contain nested lists)
+    'varargs':  character
+    'keywords': character
+    'defaults': character
```

Comment:
'(it may contain nested lists)' don't know what this means.   suggest omitting.
I would link the Python inspect.getargspec module.